src/picturecommands.py

```diff
@@ -445,6 +454,284 @@ def __init__(self, bot):
 
         self.sent_messages_image_urls = dict()
 
+    async def image_command_error(self, ctx, user_input, syntax):
+        embed = discord.Embed(
+            title="Uh oh!",
+            description=f"I had trouble understanding that. `{user_input.capitalize()}` doesn't seem to be a command...",
+            color=discord.Color.dark_red()
+        )
+        all_invocations = get_all_cmds_aliases_from_db(
+            self.bot.db_connection)
+        most_similar = process.extractOne(
+            user_input, all_invocations)
+        # For some reason, I have to add these manually or else they don't show up
```

Comment:
I wasn't able to add the footer/syntax field within a single constructor statement, I had to add them in their own lines, which didn't have a clear reason. I just did it this way because it was faster.